src/repl/scala/tools/nsc/interpreter/MemberHandlers.scala

```diff
@@ -214,27 +214,34 @@ trait MemberHandlers {
     val Import(expr, selectors) = imp
     def targetType = intp.global.rootMirror.getModuleIfDefined("" + expr) match {
       case NoSymbol => intp.typeOfExpression("" + expr)
-      case sym      => sym.thisType
+      case sym      => sym.moduleClass.thisType
     }
-    private def importableTargetMembers = importableMembers(targetType).toList
+    private def importableTargetMembers = importableMembers(exitingTyper(targetType)).toList
```

Comment:
Does this PR introduce any new call-sites of `importableTargetMembers` that aren't already in `exitingTyper`?

Currently, IntelliJ says:
```
    lazy val individualSymbols: List[Symbol] = exitingTyper(importableTargetMembers filter (m => selectorNames(m.name)))
    lazy val wildcardSymbols: List[Symbol]   = exitingTyper(if (importsWildcard) importableTargetMembers else Nil)
```